scripts/host-check

```diff
@@ -1568,6 +1720,10 @@ EXTRA_CHECKS_MAP: Mapping[str, List[Check]] = {
     "docker": DOCKER_CHECKS,
     "xr-compose": XR_COMPOSE_CHECKS,
 }
+PLATFORM_AWS_CHECKS_MAP: Mapping[str, List[Check]] = {
+    "xrd-control-plane-aws": CONTROL_PLANE_AWS_CHECKS,
```

Comment:
My comment on this seems to have gotten lost - Unless I've missed something, I don't see any value in suffixing with `-aws`, it just makes it more faff when looking up (as just cannot pass platform)

Reply:
Currently the UT assumes unique keys across all these mappings. My thoughts was this is no-harm way of complying with this restriction.